src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -659,7 +659,9 @@ private void BuildBufferedWritingScope(Chunk htmlAttributeChunk, bool htmlEncode
                 // Scopes are a runtime feature.
                 if (!_designTimeMode)
                 {
-                    _writer.WriteMethodInvocation(_tagHelperContext.StartTagHelperWritingScopeMethodName);
+                    _writer.WriteMethodInvocation(
+                        _tagHelperContext.StartTagHelperWritingScopeMethodName,
+                        "encoder: null");
```

Comment:
There's no other overload of the `RazorPage.StartTagHelperWritingScope()` method. It's only called from generated code (though through a delegate in its only other generated use). Why would a user call that `RazorPage` method?

In any case, I was just talking about changing the generated parameter here from `encoder: null` to `null`.